applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -183,6 +185,18 @@ static int get_input_report_idx(uint8_t report_id)
 	return -ENOENT;
 }
 
+static int get_output_report_size(uint8_t report_id)
+{
```

Comment:
switch/case will allow easier maintenance
you don't need a comment if you use `sizeof(report_id)` (it becomes self-explanatory)

```
switch (report_id) {
case REPORT_ID_KEYBOARD_LEDS:
   return sizeof(report_id) + REPORT_SIZE_KEYBOARD_LEDS;
default:
   return -ENOENT;
}
```